nipype/interfaces/afni/preprocess.py

```diff
@@ -2556,6 +2556,79 @@ def _format_arg(self, name, trait_spec, value):
             return super(TCorrMap, self)._format_arg(name, trait_spec, value)
 
 
+class NetCorrInputSpec(AFNICommandInputSpec):
+    in_file = File(exists=True, argstr="-inset %s", mandatory=True)
+    in_rois = File(exists=True, argstr="-in_rois %s", mandatory=True)
+    mask = File(exists=True, argstr="-mask %s")
+    weight_ts = File(exists=True, argstr="-weight_ts %s")
+    fish_z = traits.Bool(argstr="-fish_z")
+    part_corr = traits.Bool(argstr="-part_corr")
+    ts_out = traits.Bool(argstr="-ts_out")
+    ts_label = traits.Bool(argstr="-ts_label")
+    ts_indiv = traits.Bool(argstr="-ts_indiv")
+    ts_wb_corr = traits.Bool(argstr="-ts_wb_corr")
+    ts_wb_Z = traits.Bool(argstr="-ts_wb_Z")
+    ts_wb_strlabel = traits.Bool(argstr="-ts_wb_strlabel")
+    nifti = traits.Bool(argstr="-nifti")
+    output_mask_nonnull = traits.Bool(argstr="-output_mask_nonnull")
+    push_thru_many_zeros = traits.Bool(argstr="-push_thru_many_zeros")
+    ignore_LT = traits.Bool(argstr="-ignore_LT")
+    out_file = File(
+        name_template="%s_netcorr",
+        desc="output file name part",
+        argstr="-prefix %s",
+        position=1,
+        name_source="in_file",
+    )
```

Comment:
These should all have descriptions explaining what they do. And feel free to use more descriptive names than the 3dNetCorr flags.